unit-tests/src/test/java/org/eclipse/collections/impl/collector/Collectors2Test.java

```diff
@@ -152,265 +171,269 @@ public void makeString3Parallel()
     @Test
     public void toList()
     {
-        Assert.assertEquals(
-                SMALL_INTERVAL.toList(),
-                this.smallData.stream().collect(Collectors2.toList()));
-        Assert.assertEquals(
-                SMALL_INTERVAL.reduceInPlace(Collectors2.toList()),
-                this.smallData.stream().collect(Collectors2.toList()));
+        MutableList<Integer> expected = SMALL_INTERVAL.toList();
+        MutableList<Integer> actual = this.smallData.stream().collect(Collectors2.toList());
+        Assert.assertEquals(expected, actual);
+        MutableList<Integer> expected1 = SMALL_INTERVAL.reduceInPlace(Collectors2.toList());
+        Assert.assertEquals(expected1, actual);
```

Comment:
This is a nice pure refactoring, which uncovered some awkwardness. I think we should rename `expected1` to `actual2`, and have the second assertion compare `expected` to `actual2`. I'm happy to merge without doing this, as your refactoring hasn't changed behavior but wanted your opinion on whether this would make the code clearer.

Reply:
I see the same pattern across a few methods in this class. Will make changes to renames these variables in a separate commit to this PR.